pkg/asyncdr/asyncdr.go

```diff
@@ -281,6 +282,117 @@ func CreateMigrationSchedule(
 	return migSched, err
 }
 
+func CreateSnapshotSchedule(
+	namespace string,
+	pvcName string,
+	scheduleName string,
+	policyName string,
+	snapshotType string,) (*storkapi.VolumeSnapshotSchedule, error) {
+
+	snapSched := &storkapi.VolumeSnapshotSchedule{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      scheduleName,
+			Namespace: namespace,
+			Annotations: map[string]string{
+				"portworx/snapshot-type": snapshotType,
+			},
+		},
+		Spec: storkapi.VolumeSnapshotScheduleSpec{
+			Template: storkapi.VolumeSnapshotTemplateSpec{
+				Spec: crdv1.VolumeSnapshotSpec{
+					PersistentVolumeClaimName: pvcName},
+			},
+			SchedulePolicyName: policyName,
+		},
+	}
+	snapSchedule, err := storkops.Instance().CreateSnapshotSchedule(snapSched)
+	return snapSchedule, err
+}
+
+func CreateSchedulePolicyWithRetain(policyName string, interval int, retain storkapi.Retain) (pol *storkapi.SchedulePolicy, err error) {
```

Comment:
TODO later: Just have a  generic function called `CreateSchedulePolicy` and based on if `retain != nil` , use it to create SchedulePolicy with or without `retain`.
In my opinion we don't need a function just to create a snapshot will retain policy.

Reply:
Yes, as there are lot of tests using it, I will refactor this later.